components/GraphDiagram/GraphDiagram.tsx

```diff
@@ -383,18 +383,20 @@ async function colorizeGraph(svg: SVGSVGElement, colorize: string) {
 
   if (!colorizer) {
     // Unset all node colors
-    $(svg, 'g.node path').attr('style', undefined);
+    for (const node of svg.querySelectorAll('g.node path')) {
+      node.removeAttribute('style');
+    }
```

Comment:
One thing to be aware of is that the previous code silently supported multiple elements, while $$ does not. That is `$('a').remove()` would remove all `a` elements, but that requires an explicit loop now.

I think this was the only instance that needed it.